docs/release-cycle.md

```diff
@@ -17,65 +16,62 @@ That said, nothing here is set in stone. See this as inspiration :)
-   - Make sure that these all are in the `release-1.X-blocking` section of the testgrid.
-     - This is really important, as this is the section/tab the release team is monitoring and basing go/no-go decisions on
- - Open a tracking issue in `kubernetes/kubernetes` where you tag the release team, add the v1.X milestone, and `priority/critical-urgent` and `status/approved-for-milestone` labels
-   - This is needed so the release team has a clear escalation path for getting the latest information about the state of kubeadm, where release blockers, etc. can be discussed.
-   - [Example](https://github.com/kubernetes/kubernetes/issues/51841)
-   - Attend the Release Burndown Meetings whenever possible and/or communicate with the release team often to be on top of the release events.
- - Make sure all critical bugfixes get the v1.X milestone, `status/approved-for-milestone` label and merged in time.
- - Marketing & announcements
-   - Write a blog post on new features, possibly for inclusion in the "Five days of Kubernetes v1.X" blog post series on https://blog.kubernetes.io
-   - Demonstrate new features at the [Kubernetes Community Meeting](https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit)
-     and do the SIG Update on behalf of SIG Cluster Lifecycle
-   - Sync with Developer Advocates and collaborate on writing about kubeadm features, also those who aren't the most highlighted one, but still useful
-   - Write v1.X user-facing documentation in the `kubernetes/kubernetes.github.io` repo
- - **IMPORTANT**: Right before `v1.X.0-rc.1` is cut, bump the default Kubernetes version to use in kubeadm, like [this](https://github.com/kubernetes/kubernetes/pull/47440)
- - Until pushing of debs/rpms is an automated process, ping a Googler to push newly generated debs and rpms to the respective repos, can be verified via these links:
-   - debs: https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages
-   - rpms: https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64/repodata/primary.xml
+- This month; code freeze is applied and no feature PRs may merge, only critical bug fixes
+  - Right before code freeze you should ensure that all feature PRs for kubeadm are reviewed, LGTM'd and approved.
+- Attend the Release Burndown Meetings whenever possible and/or communicate with the release team often to be on top of the release events.
+- Marketing & announcements (optional)
+  - Write a blog post on new features, possibly for inclusion in the "Five days of Kubernetes v1.X" blog post series on https://blog.kubernetes.io
+  - Demonstrate new features at the Kubernetes Community Meeting.
+  - Sync with Developer Advocates and collaborate on writing about kubeadm features, also those who aren't the most highlighted one, but still useful
```

Comment:
Is it possible/advisable to document how to contact dev advocates?

Reply:
i don't know how to contact them.

Reply:
deleted this sentence.